salesforce/my-conservation-life/force-app/main/default/lwc/utils/connection.js

```diff
@@ -1,3 +1,3 @@
-const URL = 'https://my-conservation-life.herokuapp.com/api/v1/';
+const URL = 'https://my-conservation-life-dev2.herokuapp.com/api/v1/';
```

Comment:
This should be reverted before going to `dev`

Reply:
yes